### DISCUSSION TITLE
Created collection, but on the front end the content from the index.antlers.html file is not showing

### DISCUSSION THREAD

**Discussion by eduard-un** [statamic/cms#6211] [Q&A]
Instead of seeing the HTML code, I put into `views/blog/index.antlers.html` file I get nothing on the front end: 
![Screenshoot - Code - index antlers html — divitech (Workspace)](https://user-images.githubusercontent.com/1719735/173627520-76d51180-28d3-43bc-a1aa-6a103b1d1de5.jpg)

What am I missing? 

This is the setup of my "Blog Page": 
![Screenshoot - Google Chrome - Blog ‹ Pages ‹ Collections ‹ Statamic](https://user-images.githubusercontent.com/1719735/173627688-929400db-8273-45dd-9e9b-7dcba40158f1.jpg)

And this is my Blog Collection setup: 
![Screenshoot - Google Chrome - Configure Collection ‹ Statamic](https://user-images.githubusercontent.com/1719735/173627786-bff609b7-6059-46e0-a160-f932b0954784.jpg)

I was trying to follow the steps in this intro video: https://www.youtube.com/watch?v=WtelkWCDqE4&list=PLVZTm2PNrzMwYLGotkQvTvjsXAkANJIkc&index=6 

However, nothing works on my end. 

Any input would be much apreciated

**Comment by jackmcdade** [comment]
Does your layout file have `{{ template_content }}` in there?

**Reply by eduard-un** [reply]
Yes it has, the homepage template shows my custom HTML code as expected.

**Comment by jasonvarga** [comment]
Your blog entry is in the `pages` collection. What's the `route` in there? (Not the `blog` collection)

**Reply by eduard-un** [reply]
Do you mean this? 
![Screenshoot - Google Chrome - Blog ‹ Pages ‹ Collections ‹ Statamic](https://user-images.githubusercontent.com/1719735/173771638-1d2bb956-49d5-4ae5-9749-866125cc66d9.jpg)

Other than that, I don't have an option to set a `route` for my Blog Page.

**Reply by jasonvarga** [reply]
No, I mean the edit the Pages collection.

In your original post you said

> And this is my Blog Collection setup:

But you should check the route of your Pages collection.

Or, check `content/collections/pages.yaml`.
